### PR TITLE
Close activity command animated

### PR DESCRIPTION
**Pull request by Jmvcaseiro**

### CODE REVIEW

**Review by bruno-garces — APPROVED** [review]

**Review by neteinstein — APPROVED** [review]

File: base/src/main/java/com/mindera/skeletoid/routing/CloseActivityCommand.java [hunk 1]
```diff
@@ -8,13 +8,24 @@
 public class CloseActivityCommand implements IRouteCommand {
 
     private final Activity mActivity;
```
Comment (neteinstein) [inline base/src/main/java/com/mindera/skeletoid/routing/CloseActivityCommand.java]: Maybe convert this to Kotlin and drop hungarian notation? :-)

**Review by neteinstein — APPROVED** [review]

File: base/src/main/java/com/mindera/skeletoid/routing/CloseActivityCommand.java (hunk 1 above)
Comment (Jmvcaseiro) [reply base/src/main/java/com/mindera/skeletoid/routing/CloseActivityCommand.java]: Right away boss! I thought we weren't using kotlin for some reason. I even commented that with @bruno-garces

Comment (neteinstein) [reply base/src/main/java/com/mindera/skeletoid/routing/CloseActivityCommand.java]: The only reason is that we started this way back.. when Kotlin wasn't official. 
I think in the mid-term we can even merge base with kotlin extensions modules, and use everything in Kotlin.

:-)

**Comment by Jmvcaseiro** [comment]
> if for some reason we need it because it is being accessed from a java class, we can add the annotation @JvmOverloads to cover that
-> it was indeed that. Good to know!

**Review by ritatferreira — APPROVED** [review]

**Review by djrsousa — APPROVED** [review]

**Pull request merged**